FreshBuy/pages/cart/cart.html

```diff
@@ -0,0 +1,152 @@
+                        </div>
+                        <h4 class="product-price">Rs.50 <span class="original-price text-strike-through">Rs.60</span><span class="discount-percentage">20% off</span></h4>
+                        <span class="qty-scale text-xl">Quantity:
+                            <button class="inc">+</button>
+                            <input type="number" id="qty" readonly="" value="0">
+                            <button class="dec">-</button>
+                        </span>
+                        <button class="btn btn-solid-primary" >Move to Wishlist</button>
+                        <button class="btn btn-outline-primary" >Remove</button>
+                    </div>
+                </div>
+                <div class="card-container horizontal">
+                    <div class="card-img horizontal-img border-right">
+                        <img src="../../assets/images/fruits/orange.png" alt="Apple">
+                    </div>
+                    <div class="card-details card-details-horizontal">
+                        <h2 class="card-heading">Orange<span><i class="far fa-heart"></i></span></h2>
+                        <p class="card-sub-heading">Fresh orange from farm</p>
+                        <div class="rating text-lg">
+                            <span class="rating-value">4.3<i class="fa fa-star checked margin-l"></i></span>
+                            (<span class="rating-number">2333</span>)
+                        </div>
+                        <h4 class="product-price">Rs.100 <span class="original-price text-strike-through">Rs.110</span><span class="discount-percentage">20% off</span></h4>
+                        <span class="qty-scale text-xl">Quantity:
+                            <button class="inc">+</button>
+                            <input type="number" id="qty" readonly="" value="0">
+                            <button class="dec">-</button>
+                        </span>
+                        <button class="btn btn-solid-primary" >Move to Wishlist</button>
+                        <button class="btn btn-outline-primary" >Remove</button>
+                    </div>
+                </div>
+            </section>
+            <section class="item-data margin-t">
+                <h3 class="text-3xl">Price Details</h3>
+                <hr>
+                <p>Price <span class="rate">Rs.210</span></p>
+                <p>Discount <span class="rate">-Rs.20</span></p>
+                <p>Delivery Charge <span class="rate">Rs.40</span></p>
+                <hr>
+                <h3 class="text-2xl">Total Amount <span class="rate">Rs.230</span></h3>
+                <hr>
+                <p>You will save Rs.20 on this order</p>
+                <button class="btn btn-outline-primary margin-t" >Place Order</button>
+            </section>
+        </div>
+    </div>
+    <script>
+        const qty = document.querySelector("#qty");
```

Comment:
You can separate the JavaScript into separate file

Reply:
ok, I wil do that